examples/validator/Cargo.toml

```diff
@@ -5,9 +5,12 @@ publish = false
 version = "0.1.0"
 
 [dependencies]
-anyhow = "1.0.44"
-axum = {path = "../.."}
-serde = {version = "1.0", features = ["derive"]}
+async-trait = "0.1"
+axum = { path = "../.." }
+serde = { version = "1.0", features = ["derive"] }
 thiserror = "1.0.29"
-tokio = {version = "1.0", features = ["full"]}
-validator = {version = "0.14.0", features = ["derive"]}
+tokio = { version = "1.0", features = ["full"] }
+validator = { version = "0.14.0", features = ["derive"] }
+tracing = "0.1"
+tracing-subscriber = "0.2"
+http-body = "0.4.3"
```

Comment:
Now there's a final newline missing here 😄

Also I think it would be nice if the dependencies were sorted alphabetically.